yaml-tests/src/test/resources/prepared.yamsql

```diff
@@ -19,7 +19,7 @@
 
 ---
 options:
-    supported_version: 4.1.6.0
+    supported_version: 4.0.559.6
```

Comment:
I feel like there should be a comment here, because it doesn't work on anything between `4.0.559.6` and `4.1.6.0`.
Maybe something like:
```suggestion
    # This does not work against any version between 4.0.559.6 and 4.1.6.0 because server support was backported
    # to a patch release. We would like to surface that in the mixed mode test results rather than hide it.
    supported_version: 4.0.559.6 # this does not actually work 
```

Reply:
Done.